setup.py

```diff
@@ -172,16 +169,19 @@ def run(self):
         sdist.run(self)
 
 
+install_requires = ["numpy", "pillow"]
+
 extras_require = {
+    "linting": ["black", "flake8"],
+    "dev": ["invoke", "pytest", "pytest-cov"],
+    "docs": ["sphinx", "numpydoc"],
+    "ffmpeg": ["imageio-ffmpeg", "psutil"],
```

Comment:
Nice catch - I'd assumed psutil was needed by the tests

Reply:
you are right, too. `psutil` is needed exclusively for the tests of `imageio-ffmpeg`. I didn't want to introduce another `test-ffmpeg` target, so I combined it with `ffmpeg`.